simplic-user/Simplic.User.UI/Converters/BoolToVisibilityConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Simplic.User.UI
+{
+    class BoolToVisibilityConverter : IValueConverter
```

Comment:
If we use this converter more often, we can move it to some core libs as well.

Reply:
Done